docs/Using-Fleet/Permissions.md

```diff
@@ -78,6 +78,7 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | View results of MDM commands executed on macOS hosts enrolled in Fleet's MDM                                                               | ✅        | ✅         | ✅          | ✅     |        |
 | Edit [MDM settings](https://fleetdm.com/docs/using-fleet/mdm-macos-settings)                                                               |          |           |            | ✅     | ✅      |
 | Edit [MDM settings for teams](https://fleetdm.com/docs/using-fleet/mdm-macos-settings)                                                     |          |           |            | ✅     | ✅      |
+| Upload an EULA file for MDM automatic enrollment\*                                                                                         |          |           |            | ✅     |         |
```

Comment:
Super tricky, but this should be "Upload a EULA file..." because even though EULA starts with a vowel, it makes the sound of a consonant (Y sound).